models/src/main/thrift/content/v1.thrift

```diff
@@ -1147,3 +1147,191 @@ struct Package {
     /* The package name */
     3: required string packageName
 }
+
+struct ContentStats {
```

Comment:
I don't think this is needed? You're not using it anywhere and it resides within content so should be fine to delete.